src/components/form-control/__tests__/__snapshots__/form-control.test.js.snap

```diff
@@ -0,0 +1,143 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Should we use snapshots for these? In my experience, over time it could get tedious to review snapshots. It looks like most of these are very similar in structure as well, which would require clear context to review (i.e. what are `correct props` and `correct error value`).

Would it be clearer to just write assertion for the specific case we're catching?